lib/modules/manager/leiningen/extract.spec.ts

```diff
@@ -124,6 +136,81 @@ describe('modules/manager/leiningen/extract', () => {
         },
       ],
     });
+
+    expect(extractPackageFile(leinParentProjectClj)).toMatchSnapshot({
```

Comment:
```suggestion
    expect(extractPackageFile(leinParentProjectClj)).toObject({
```

Reply:
This gives me
```
    TypeError: expect(...).toObject is not a function

      146 |     });
      147 |
    > 148 |     expect(extractPackageFile(Fixtures.get(`parent.clj`))).toObject({
          |                                                            ^
      149 |       deps: [
      150 |         {
      151 |           depName: 'org.clojure:core.async',

      at Object.<anonymous> (lib/modules/manager/leiningen/extract.spec.ts:148:60)
```

Reply:
ups, typo. updated suggestion.

Reply:
https://github.com/renovatebot/renovate/pull/29552/commits/3dbd1f7367dbd2f3af1e13a1b13df517ba9bc7ff